storybook/src/utils/Gap/Gap.tsx

```diff
@@ -0,0 +1,17 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import { HTMLAttributes, PropsWithChildren } from 'react'
+
+const gapLengthOptions = ['xs', 'sm', 'md', 'lg', 'xl'] as const
+type GapLength = (typeof gapLengthOptions)[number]
```

Comment:
Why not just specify the options here? We're not exporting `gapLengthOptions` or using it anywhere else.